sdk/communication/azure-communication-common/src/main/java/com/azure/android/communication/common/CommunicationCloudEnvironment.java

```diff
@@ -23,10 +23,6 @@ public CommunicationCloudEnvironment(String environmentValue) {
         this.environmentValue = environmentValue;
     }
 
-    static CommunicationCloudEnvironment fromModel(CommunicationCloudEnvironmentModel environmentModel) {
```

Comment:
I thought this is the preferred way of doing it. Azure review board suggested this.

Reply:
I think @minnieliu  changed it, do you want to change it back in next version or in this version?

Reply:
Let's leave it as is